server/forward.go

```diff
@@ -262,6 +263,10 @@ func forwardRegionHeartbeatToScheduling(forwardStream schedulingpb.Scheduling_Re
 			errCh <- errors.WithStack(err)
 			return
 		}
+		// TODO: find a better way to halt scheduling immediately.
+		if rc.GetOpts().IsSchedulingHalted() {
+			continue
```

Comment:
One problem is that the operator generated by the scheduling server still exists in memory. Is it possible to cancel all the scheduling server operators after the online recovery is complete so that the scheduling can be resumed as soon as possible?

Reply:
It depends. IMO, if those operators still take effect, cancel all is not the best way. But if the region is changed, we may need to cancel. So I prefer to leave it for now.